indexer-app/src/cmr/indexer/services/index_set_service.clj

```diff
@@ -280,6 +281,18 @@
 
     (index-requested-index-set context index-set es-cluster-name)))
 
+(defn put-index-set
+  "Upsert the given index-set to the index-sets index in ES."
+  [context index-set]
+  (let [split-index-set-map (split-index-set-by-cluster index-set)]
+    (acl/verify-ingest-management-permission context :update)
```

Comment:
Just curious as to why you moved the acl check in this function instead of leaving it in the routes namespace?  The other routes that have the check, check the acl in the routes namespace. The rest is a good move.